lib/mix/tasks/setup.ex

```diff
@@ -11,8 +11,7 @@ defmodule Mix.Tasks.LiveSvelte.Setup do
       "configure_phoenix",
       "configure_esbuild"
     ]
-    |> Enum.map(&Task.async(fn -> Mix.Task.run("live_svelte." <> &1) end))
-    |> Enum.map(&Task.await(&1, :infinity))
+    |> Enum.each(&Mix.Task.run("live_svelte." <> &1))
```

Comment:
Note: `Enum.each` is used over `Enum.map` because running the tasks are side-effects and we don't need anything to be returned here.